### DISCUSSION TITLE
How to use custom interface?

### DISCUSSION THREAD

**Discussion by SNPR** [directus/directus#3515] [Archive]
Hello! I followed [this](https://docs.directus.io/guides/extensions/interfaces.html#create-a-custom-interface) instruction and created custom interface:

![image](https://user-images.githubusercontent.com/22043414/103136376-a5e89080-46e1-11eb-93cf-1a64e362b4af.png)
![image](https://user-images.githubusercontent.com/22043414/103136383-b6007000-46e1-11eb-97bb-a1dfa7fccb6b.png)
![image](https://user-images.githubusercontent.com/22043414/103136387-bd277e00-46e1-11eb-8f95-2977ab88a5b2.png)

Then I installed rollup and created bundle for this interface.
Then I moved index.js (created by rollup) to the /interfaces folder.

Using API I can see my interface in the list:
GET http://127.0.0.1:8055/extensions/interfaces/
Response: 
```
{
  "data": [
    "CustomInterface"
  ]
}
```

But I can't see my newly created interface in the interfaces list. What I did wrong?

![image](https://user-images.githubusercontent.com/22043414/103136420-27d8b980-46e2-11eb-959e-5050367a03fe.png)

**Comment by SNPR** [comment]
Oh, I just figured out that I should move generated index.js not to /interfaces folder but to /interfaces/InterfaceName folder :-)

**Comment by SNPR** [comment]
I have another question - what should I do in order to save my custom component's input to the database?
My component code is:
```
<template>
  <div>
    <div>My Custom Interface</div>
    <input v-model="value" />
    <p>{{ value }}</p>
  </div>
</template>

<script>
  export default {
    props: {
      value: String
    }
  };
</script>
```

After changing input value I click save and there is no value in the database in this field.

**Reply by benhaynes** [reply]
Separate questions should be separate tickets... 🙂
